lib/charms/mongodb/v0/helpers.py

```diff
@@ -244,10 +247,19 @@ def process_pbm_status(pbm_status: str) -> StatusBase:
     if "(none)" in current_pbm_op(pbm_status):
         return ActiveStatus("")
 
-    if "Snapshot backup" in current_pbm_op(pbm_status):
-        return MaintenanceStatus("backup started/running")
+    # Example of backup id: 2023-08-21T13:08:22Z
+    backup_match = re.search(
+        r'Snapshot backup "(?P<backup_id>\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z)"', pbm_status
+    )
+    if backup_match:
+        backup_id = backup_match.group("backup_id")
+        return MaintenanceStatus(f"backup started/running, backup id:'{backup_id}'")
 
-    if "Snapshot restore" in current_pbm_op(pbm_status):
-        return MaintenanceStatus("restore started/running")
+    restore_match = re.search(
+        r'Snapshot restore "(?P<backup_id>\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z)"', pbm_status
```

Comment:
**niptick, suggestion** is this the same of line 252? you could maybe factor this out with `re.compile` in one location and then use that